input/fsh/profiles/FinnishSchedulingPatient.fsh

```diff
@@ -0,0 +1,66 @@
+Profile: FinnishSchedulingPatient
```

Comment:
There's probably nothing scheduling specific here?
We should change to use the profile from base profiles.